src/server.js

```diff
@@ -25,20 +24,14 @@ const init = new Promise((resolve, reject) => {
     const timeoutPromise = new Promise((res, rej) => {
         setTimeout(res, FIVE_SECONDS_IN_MS);
     }).then(value => LogUtils.log({ message: 'server.js: Timeout reached' }));
+
     authToken.then(() => {
         // await data
-        const dataInit = Promise.race([
+        Promise.race([
```

Comment:
why don't you need to save this `dataInit` anymore

Reply:
before it was used on line 39, but that isn't the case anymore

Reply:
^^ basically I deprecated `GhopperUtils.isGraphhopperReady`, and then there's only one Promise in the `Promise.all`, so that can be removed completely and the `then` clause can be moved into the original `then` clause of `dataInit`